test/libevmasm/EVMAssemblyTest.h

```diff
@@ -0,0 +1,58 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+
+#pragma once
+
+#include <test/TestCase.h>
+
+#include <libsolidity/interface/OptimiserSettings.h>
+
+#include <libevmasm/Assembly.h>
+
+#include <memory>
+#include <ostream>
```

Comment:
```suggestion
```
unused

Reply:
It's used on line 42:
```c++
	TestResult run(std::ostream& _stream, std::string const& _linePrefix = "", bool const _formatted = false) override;
```

Reply:
I missed that one, thanks for pointing it out! (line _42_ of all things...)